### ISSUE TITLE
[Feature] Expand Young Jump Connector

### ISSUE THREAD

**Issue by perrinaz** [manga-download/hakuneko#3023]
**Is your feature request related to a problem? Please describe.**
There is so much more available for off=line reading from the youngjump.world website.

**Describe the solution you'd like**
I would like the available chapters to include more than just the last three chapters.  When I look at my screen, I can see the current issue, the 4 previous issues, and the 4 oldest issues available (that will soon be going away).

**Describe alternatives you've considered**
I've tried using the clipboard feature on the /mypage/ link and the /backnumber/ link

**Context (please complete the following information):**
 - Your current HakuNeko Version [e.g. 6.17]
 - I do not know where to find it.  It was a nightly build 1 or 2 nights ago.

**Additional context**

**Comment by Robonau** [comment]
you mentioned that you 
> tried using the clipboard feature on the /mypage/ link and the /backnumber/ link

but didn't say if it worked or not, does it? (they will require you to login if to work (if they do work) )

the lack of support for other parts of the site are likely due to no devs having access to the paid part of the site

**Comment by perrinaz** [comment]
Oh.  I'm sorry.  I thought I had mentioned that it didn't work, but I wrote that just before going to bed.  What happened when I used the /mypage/link was getting "Manga #34C" in the Manga List.  Clicking on that "manga" gives no chapters, however.  When I put the /backpages/ link I get a similar result.  I don't remember what the "Manga #xxx" was exactly but it was a single manga titled like that with no chapters listed when selected.

**Comment by perrinaz** [comment]
I'm thinking part of the problem might be credentials.  I use the login function that the connector offers and when I use the "open" feature of the connector it shows me logged in but as I (try) to dig into what's being served when I go to one of the paid links, it's beginning to look like a redirect to the login screen.  If someone would like to tackle it, I am willing to help where I can.  Javascript, especially when it comes to sessions and credentials and such, are a little beyond my knowledge base.

**Comment by Ever4engel** [comment]
yes in this website you must login to see (theoretically) your purchased contents. we don't have one, so can't test it.

**Comment by perrinaz** [comment]
And I have the paid account so I can test.  I just don't know how to get the credentials to pass.  With other websites, like Viz/Shonen Jump, I can just open a browser window in HakuNeko, log in there, and go back and download the manga I'm looking to read.  That method doesn't work with Young Jump.  From the structure of the source, I think at least the /backnumber/ link would work if the credentials were sent, too.

**Comment by Ever4engel** [comment]
wait for #3033 PR, this should fix a wrong behavior.

**Comment by perrinaz** [comment]
So I loaded/tested PR#3033 and it looks like the login issue resolved.  However, I still don't get a list of chapters.  

Here is what the main page looks like in Firefox/Chrome:
https://i.imgur.com/o7EfAZx.png

and here's what it looks like in HakuNeko using F-12:
https://i.imgur.com/OAHJlyu.png

Also, When I click on the link to show the preview, I get a message in the command prompt where I started HakuNeko that says "[4280:0207/033002.600:ERROR:CONSOLE(0)] "Blocked script execution in 'data:text/html;....." and goes on really long.  But I only get that message when I try to "preview" the page in F-12.  So it might not have anything to do with what's going on.

**Comment by Ever4engel** [comment]
page where you get your paid contents : 

https://www.youngjump.world/mypage/
and the link for back issues is https://www.youngjump.world/backnumber/

**Comment by artsunjai** [comment]
Still can't download paid content
